third-party/jemalloc/README

```diff
@@ -0,0 +1,11 @@
+==========================
+jemalloc README for Chapel
+==========================
+
+This copy of jemalloc is being released with Chapel for convenience and
+uses a snapshot from:
```

Comment:
jemalloc is versioned too. I'll be adding [`jemalloc-4.0.4`] (https://github.com/jemalloc/jemalloc/releases)